graph/telemetry/istio/appender/kiali_annotation.go

```diff
@@ -0,0 +1,69 @@
+package appender
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph"
+	"github.com/kiali/kiali/models/annotation"
+)
+
+const AnnotationAppenderName = "kialiAnnotation"
+
+// Name: annotation
+type AnnotationAppender struct{}
+
+// Name implements Appender
+func (a AnnotationAppender) Name() string {
+	return AnnotationAppenderName
+}
+
+// AppendGraph implements Appender
+func (a AnnotationAppender) AppendGraph(trafficMap graph.TrafficMap, globalInfo *graph.AppenderGlobalInfo, namespaceInfo *graph.AppenderNamespaceInfo) {
+	if len(trafficMap) == 0 {
+		return
+	}
+
+	if getWorkloadList(namespaceInfo) == nil {
+		workloadList, err := globalInfo.Business.Workload.GetWorkloadList(namespaceInfo.Namespace)
+		graph.CheckError(err)
+		namespaceInfo.Vendor[workloadListKey] = &workloadList
+	}
+
+	a.applyAnnotationPresence(trafficMap, namespaceInfo)
+}
+
+func (a *AnnotationAppender) applyAnnotationPresence(trafficMap graph.TrafficMap, namespaceInfo *graph.AppenderNamespaceInfo) {
+	for _, n := range trafficMap {
+		// Skip the check if this node is outside the requested namespace, we limit badging to the requested namespaces
+		if n.Namespace != namespaceInfo.Namespace {
+			continue
+		}
+
+		// We whitelist istio components because they may not report telemetry using injected sidecars.
+		if config.IsIstioNamespace(n.Namespace) {
+			continue
+		}
+
+		// dead nodes tell no tales (er, have no pods)
+		if isDead, ok := n.Metadata[graph.IsDead]; ok && isDead.(bool) {
+			continue
+		}
```

Comment:
Dead nodes have no pods but I think it's possible they have workloads.  I think we should delete these lines.

Reply:
In this case we can remove them, because for workload we are fetching the annotations of their first pod, so if they have not pods so not makes sense.